DesignerWidgets/DirectCollapsibleFrame.widget

```diff
@@ -0,0 +1,47 @@
+{
+    "name":"DirectCollapsibleFrame",
+    "baseWidget":"DirectFrame",
+    "moduleName":"DirectCollapsibleFrame",
+    "displayName":"Collapsible Frame",
+    "className":"DirectCollapsibleFrame",
+    "classFilePath":"DirectGuiExtension.DirectCollapsibleFrame",
+    "customProperties":[
+        {
+            "displayName":"Header Height",
+            "internalName":"headerheight",
+            "editType":"float",
+            "postProcessFunctionName": "setCollapsed",
+            "defaultPixel": 20
+        },
+        {
+            "displayName":"Collapsed",
+            "internalName":"collapsed",
+            "editType":"bool"
+        },
+        {
+            "displayName":"Collapse Text",
+            "internalName":"collapseText",
+            "editType":"str",
+            "postProcessFunctionName": "setCollapsed"
+        },
+        {
+            "displayName":"Extend Text",
+            "internalName":"extendText",
+            "editType":"str",
+            "postProcessFunctionName": "setCollapsed"
+        },
+        {
+            "displayName":"Update Frame Size",
+            "internalName":"updateFrameSize",
+            "editType":"command",
+            "valueOptions": "updateFrameSize"
+        },
+        {
+        "internalName": "frameSize",
+        "defaultPixel": [-100, 100, -100, 100]
+        }
+    ],
+    "//addItemFunctionName":"addItem",
```

Comment:
Why have you added the slashes here and in the other widget files?

Reply:
That is an ugly way to "comment" out that line. By changing the name the loader won't recognize it and the line gets skipped. I can remove those lines if you would like. 

I also added a slash in the beginning of the DirectOptionMenu widget definition file to make sure it isn't loaded. This is because it would otherwise replace the default OptionMenu from directGui in the ToolBox panel. Meaning that the user could no longer create the default OptionMenu. I think we talked a bit about that issue earlier, but it would require some changes to DirectGuiDesigner to fix (the designer does not expect several different widgets to have the same name).

Reply:
I thought so. Well, If there is a reason to keep them in the file, I'm probably fine with that, otherwise I'd prefer to just get rid of those lines.
For the option menu, yeah, I remember talking about that. We should open up an issue for that on the designer to not forget about it.